### PR TITLE
Routing correcting

### PR DESCRIPTION
**Pull request by Wade-Coplen**
# Description
Created a passing test that check if the landing page renders without crashing
Small correction to ensure the landing page is going to the signin/signup page.
Created link from the landing page to Signup and Signin
Added a link to go back to main page
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [x ] Test A
- [ ] Test B

# Checklist

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x ] There are no merge conflicts

### CODE REVIEW

**Review by Carrillo9Ana — APPROVED** [review]
looks good

File: src/App.js [hunk 1]
```diff
@@ -4,13 +4,23 @@ import { Fragment } from 'react';
...
+import LoginForm from './components/LoginForm';
+import TaskPage from './components/TaskPage';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 const App = () => {
   return (
     <Fragment>
-      <Route exact path="/signup" component={SignUpForm} />
-      <PrivateRoute exact path="/dashboard" component={Dashboard} />
+      <Router>
```
Comment (jgarrow) [inline src/App.js:15]: The `development` branch has `App` wrapped in the `Router` up a level in the `index.js` file, so you don't need to wrap it in a `Router` again here

File: src/components/LandingPage.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 
 const LandingPage = () => {
   return (
     <div>
       <h1>Welcome to</h1>
       <img src="OrangeLogo.png" alt="orange squares icon" />
-      <button>Sign Up!</button>
-      <button>Sign In!</button>
+
+      <Link to="/SignUp">
```
Comment (jgarrow) [inline src/components/LandingPage.js:11]: 🙌

**Review by jgarrow — CHANGES_REQUESTED** [review]
I would pull from the `development` branch and merge any conflicts first since it's been a while. And we don't need to wrap everything in a `Router` in the `App.js` file since that component is already wrapped in it in the `index.js` file. But other than that, I think it looks good!

**Review by JaxAtwood — APPROVED** [review]

**Review by davidmoates — APPROVED** [review]

File: src/components/LandingPage.js (hunk 2 above)
Comment (jgarrow) [inline src/components/LandingPage.js:11]: Capitalization of this url doesn't match the one in the Route in App.js

**Review by jgarrow — CHANGES_REQUESTED** [review]
The `to` path in `LandingPage.js` doesn't match the one in `App.js` for the signup

**Review by tinomen — COMMENTED** [review]
OK

**Pull request merged**